.github/actions/display-jest-test-summary/action.yml

```diff
@@ -0,0 +1,32 @@
+name: Display E2E Test Summary
```

Comment:
Minor, but may be better to label this "Jest" instead. The step in the workflows can overwrite this name if we want to more specific (such as E2E test summary).

Reply:
Oh yes sorry, this is what it's called in the recalls repo